client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -0,0 +1,384 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { WPCOM_FEATURES_PREMIUM_THEMES } from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import { useStarterDesignsQuery } from '@automattic/data-stores';
+import {
+	UnifiedDesignPicker,
+	PremiumBadge,
+	useCategorization,
+	getDesignPreviewUrl,
+} from '@automattic/design-picker';
+import { useLocale } from '@automattic/i18n-utils';
+import { StepContainer } from '@automattic/onboarding';
+import { useViewportMatch } from '@wordpress/compose';
+import { useSelect, useDispatch } from '@wordpress/data';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState, useEffect } from 'react';
+import FormattedHeader from 'calypso/components/formatted-header';
+import WebPreview from 'calypso/components/web-preview/content';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { urlToSlug } from 'calypso/lib/url';
+import { useSite } from '../../../../hooks/use-site';
+import { useSiteIdParam } from '../../../../hooks/use-site-id-param';
+import { useSiteSlugParam } from '../../../../hooks/use-site-slug-param';
+import useTrackScrollPageFromTop from '../../../../hooks/use-track-scroll-page-from-top';
+import { ONBOARD_STORE, SITE_STORE } from '../../../../stores';
+import { getCategorizationOptions } from './categories';
+import { STEP_NAME } from './constants';
+import PreviewToolbar from './preview-toolbar';
+import UpgradeModal from './upgrade-modal';
+import type { Step, ProvidedDependencies } from '../../types';
+import './style.scss';
+import type { Design } from '@automattic/design-picker';
+
+/**
+ * The site setup design picker
```

Comment:
```suggestion
 * The unified design picker
```

🙈 or just delete the comment :')